routes/matched_friends.js

```diff
@@ -35,4 +35,18 @@ var connection = mysql.createConnection({
     database: 'crypto_db'   
 });
 
+var id = 1;
+
+
+router.get("/matchedFriends/feed", function(req, res) {
+    connection.query(
+      "SELECT users_purchases.date_purchased, users_purchases.amount, deals.deal_name, users.username, users_profiles.photo, venues.venue_name, crypto_metadata.crypto_symbol AS crypto_symbol FROM users_purchases LEFT JOIN deals ON users_purchases.deal_id = deals.id LEFT JOIN users ON users_purchases.user_id = users.id LEFT JOIN crypto_info ON users_purchases.crypto_id = crypto_info.id LEFT JOIN crypto_metadata ON crypto_metadata_name = crypto_metadata.crypto_name LEFT JOIN venues ON venue_id = venues.id LEFT JOIN users_profiles ON users_profiles.user_id = users.id WHERE payment_received = ? AND NOT permission = ? AND users.id IN (SELECT matched_friend_id AS id FROM users_matched_friends WHERE user_id = ? AND both_accepted = 1) ORDER BY users_purchases.date_purchased DESC",
```

Comment:
"AND NOT permission = ? AND users.id IN (SELECT matched_friend_id AS id FROM users_matched_friends WHERE user_id = ? AND both_accepted = 1)"

Can you explain this part @avakrishn?

Reply:
@simonnguyen3054 This is a nested SQL query that allows me to select all the rows from the table where the users id's are all the matched friends.